src/commands.js

```diff
@@ -24,29 +18,28 @@ export class AddColonyPointsCommand {
 
   toDict() {
     return {
-        name: "AddColonyPointsCommand",
-        points: this._points
+      name: "AddColonyPointsCommand",
+      points: this._points
     };
   }
 
-  static fromDict(production_sheet, dict) {
+  static fromDict(production_sheet, _data, dict) {
```

Comment:
What is this `_data` used for?

Reply:
To make the CommandFactory more "generic", I had to make sure all the `fromDict` methods in each of the command classes has the same signature:  `production_sheet`, `data` (which is the loaded stored data, including ships/techs that may not have been loaded into the production sheet on a refresh yet), and `dict` (which is the object-specific stored data).

Because some of the `fromDict` instances don't need the `data` instance, but it still needs to be in the signature so they are all the same, a common convention is to prepend an underscore (ie, `_data`), which basically means "we know we aren't using this variable, but we need it as part of the signature".